packages/internal/src/paths.ts

```diff
@@ -112,6 +111,8 @@ export const resolveFile = (
  */
 export const getPaths = (BASE_DIR: string = getBaseDir()): Paths => {
   const routes = resolveFile(path.join(BASE_DIR, PATH_WEB_ROUTES)) as string
+  const { schemaPath } = getConfig(getConfigPath()).api
+  const schemaDir = path.parse(schemaPath).dir
```

Comment:
You can also use `path.dirname`
```suggestion
  const schemaDir = path.dirname(schemaPath)
```